pom.xml

```diff
@@ -103,7 +103,7 @@ flexible messaging model and an intuitive client API.</description>
     <netty.version>4.1.51.Final</netty.version>
     <netty-tc-native.version>2.0.30.Final</netty-tc-native.version>
     <storm.version>2.0.0</storm.version>
-    <jetty.version>9.4.29.v20200521</jetty.version>
+    <jetty.version>9.4.30.v20200611</jetty.version>
```

Comment:
Latest versio is 9.4.31.v20200723

Reply:
Removed this because a parallel change already upgraded jetty